test/edu/csus/ecs/pc2/core/util/AbstractTestCase.java

```diff
@@ -792,6 +792,22 @@ public void editFile(String filename) {
         }
     }
     
+    public void editFile(String filename, String comment) {
+        
+        System.err.println("editFile: "+comment+" file: "+filename);
+
+        String viFile = findViExecutable();
```

Comment:
Unless I missed something, from this point onward to the end of this (new) method, the code in this method is identical to the existing method `editFile(String filename)`.  Why add a duplicate set of code instead of just calling that existing method?